__tests__/newsApi.test.js

```diff
@@ -132,4 +120,38 @@ describe('CORE Task 4: GET request /api/articles/:article_id', () => {
                 expect(res.body.msg).toBe('400 Invalid Input')
                })
     });
+});
+
+describe('CORE Task 5: GET request /api/articles', () => {
+  test('Status 200: GET request fetches all articles in the array and checks if the array by date is in descending order', () => {
+    return request(app)
+           .get('/api/articles')
+           .expect(200)
+           .then((res)=>{
+            const articlesArray = res.body.articles 
+            expect(articlesArray).toHaveLength(13)
+            articlesArray.forEach((article)=>{
+              expect(article).toMatchObject({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(String),
+              })
+              expect(article).not.toHaveProperty('body')
+            })
+            expect(articlesArray).toBeSortedBy('created_at',{descending: true})
+           })
+  });
+  test('Status 404: GET request with an endpoint /api/arthicles misspelt results in error message', () => {
+    return request(app)
+           .get('/api/arthicles')
+           .expect(404)
+           .then((res)=>{
+            expect(res.body.msg).toBe('404 Route Not Found')
+           })
+  });
```

Comment:
This test isn't per se wrong, but you already tested for this bit of functionality in Core Task 2 with `/api/toppic`